setup.py

```diff
@@ -66,9 +90,31 @@ def call(args, cwd=ABS_PATH):
 include_dirs=[
     './',
     './src',
-    './src/ccan',
     './src/secp256k1/include',
     ]
+library_dirs = [
+    ]
+libraries = [
+    ]
+sources = [
+    'src/swig_python/swig_python_wrap.c',
+    ]
```

Comment:
I formatted them this way so that future additions would only need to insert a new line rather than delete and insert. It's the same reason as for putting a comma after the last item in a list, which is a practice that was already applied to this file before I got to it.